doc/usage.rst

```diff
@@ -64,191 +63,41 @@ Config file:
-    (1473, 9) (1473,)
-    >>> print(categorical)
-    [False, True, True, False, True, True, True, True, True]
 
-In case you are working with `scikit-learn
-<http://scikit-learn>`_, you can use this data right away:
+~~~~~~~~~~~~
+Key concepts
+~~~~~~~~~~~~
 
-.. code:: python
-
-    >>> from sklearn import preprocessing, ensemble
-    >>> enc = preprocessing.OneHotEncoder(categorical_features=categorical)
-    >>> print(enc)
-    OneHotEncoder(categorical_features=[False, True, True, False, True, True, True, True, True],
-           dtype=<class 'numpy.float64'>, handle_unknown='error',
-           n_values='auto', sparse=True)
-    >>> X = enc.fit_transform(X).todense()
-    >>> clf = ensemble.RandomForestClassifier()
-    >>> clf.fit(X, y)
-    RandomForestClassifier(bootstrap=True, class_weight=None, criterion='gini',
-                max_depth=None, max_features='auto', max_leaf_nodes=None,
-                min_impurity_split=1e-07, min_samples_leaf=1,
-                min_samples_split=2, min_weight_fraction_leaf=0.0,
-                n_estimators=10, n_jobs=1, oob_score=False, random_state=None,
-                verbose=0, warm_start=False)
-
-When you have to retrieve several datasets, you can use the convenience function
-:meth:`openml.datasets.get_datasets()`, which downloads all datasets given by
-a list of IDs:
-
-    >>> ids = [12, 14, 16, 18, 20, 22]
-    >>> datasets = openml.datasets.get_datasets(ids)
-    >>> print(datasets[0].name)
-    mfeat-factors
+OpenML contains several key concepts which it needs to make machine learning
+research shareable. A machine learning experiment consists of several runs,
+which describe the performance of an algorithm (called a flow in OpenML) on a
+task. Task is the combination of a dataset, a split and an evaluation metric. In
+this user guide we will go through listing and exploring existing tasks to
+actually running machine learning algorithms on them. In a further user guide
+we will examine how to search through datasets in order to curate a list of
+tasks.
 
 ~~~~~~~~~~~~~~~~~~
 Working with tasks
 ~~~~~~~~~~~~~~~~~~
 
-#TODO put a link to the OpenML documentation here! Link the Task functions and
-the task class
-
-While datasets provide the most basic information for a machine learning task,
-they do not provide enough information for a reproducible machine learning
-experiment. A task defines how to split the dataset into a train and test set,
-whether to use several disjoint train and test splits (cross-validation) and
-whether this should be repeated several times. Also, the task defines a target
-metric for which a flow should be optimized.
+Tasks are containers, defining how to split the dataset into a train and test
+set, whether to use several disjoint train and test splits (cross-validation)
+and whether this should be repeated several times. Also, the task defines a
+target metric for which a flow should be optimized. You can think of a task as
```

Comment:
I would make the "You can" sentence the first sentence. I think more essential is that the task defines which dataset to use, which column (if any) is the target and whether it's a classification, regression, clustering etc task.